pkg/kubelet/cm/cpumanager/cpu_assignment_test.go

```diff
@@ -21,11 +21,15 @@ import (
 	"sort"
 	"testing"
 
+	"k8s.io/klog/v2/ktesting"
+	_ "k8s.io/klog/v2/ktesting/init"
+
 	"k8s.io/kubernetes/pkg/kubelet/cm/cpumanager/topology"
 	"k8s.io/utils/cpuset"
 )
 
 func TestCPUAccumulatorFreeSockets(t *testing.T) {
+	logger, _ := ktesting.NewTestContext(t)
```

Comment:
I have seen the use of `ktesting.Init(t)` for creation of logger for testing. Do we know why we use `NewTestContext(t)` vs `Init(t) `?

Reply:
`ktesting.Init(t)` returns only a context and `ktesting.NewTestContext(t)` [calls `ktesting.Init(t)`](https://github.com/kubernetes/kubernetes/blob/master/test/utils/ktesting/ktesting.go#L45-L48) **and** returns 
 a context and a logger. As we need a logger here, it's shorter to call `ktesting.Init(t)`